src/card/card.jsx

```diff
@@ -12,12 +12,46 @@ const Card = React.createClass({
     };
   },
 
+  getDefaultProps() {
```

Comment:
Not until #2585 is merged. 

But if you have that planned we can ignore this for now :D